doc/users/plotting/examples/colormap_normalizations_arbitrarynorm.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
I have now added  and commited the docstring to the examples, however I have not yet moved it to the examples folder, as I am not sure which subfolder is the most appropriate, or whether I should create a new subfolder.